README.md

```diff
@@ -4,6 +4,8 @@ Fetch your NYT Crossword Puzzle solve stats and export them as CSV.
 
 The NYT app shows minimal information around your streaks and average solve times. This gets you the raw data so you can do your own analysis.
 
+This version is a departure from its predecessor and uses new API endpoints published by the NYT. The data it produces does not line up tidily with the previous version.
```

Comment:
Appreciate the disclaimer, but probably don't need this. We can tag a new release for this if anyone is curious about going back. There's really no guarantees or versioning around the schema put out by this library, so totally fine changing it up like this

```suggestion
```